test/correctness/simd_op_check.cpp

```diff
@@ -41,6 +41,26 @@ int my_process_id = 0;
 // width and height of test images
 const int W = 256*3, H = 100;
 
+bool can_run_code() {
+
+  // If we can (target matches host), run the error checking Func.
```

Comment:
On Thu, Feb 18, 2016 at 9:52 PM, Dillon Sharlet <notifications@github.com>
wrote:

> We have a clang-format file at the root of the repo that might be useful
> for this. I don't think I've actually seen an official coding convention
> myself, I just try to mirror what I see in existing code :)
>

Now that we're getting more contributors, I'll repeat a suggestion that was
made a while back: we should consider just defining .clang-format file as
canonical, reformat all our code with it, and then require all submissions
to match that format (even setting up a Golang-style presubmit checker if
feasible).

It's not a big deal in the scheme of thing but it eliminates a surprising
amount of noise from these things :-)